lms/static/sass/shared/_course_object.scss

```diff
@@ -180,78 +191,116 @@
         }
       // }
 
-      .info {
-        background: $content-wrapper-bg;
-        height: 220px + 130px;
-        left: 0px;
-        position: absolute;
-        top: 0px;
-        @include transition(all 0.15s linear 0s);
-        width: 100%;
-        overflow: hidden;
-
+      .course-image {
         .cover-image {
-          height: 200px;
+          height: 160px;
```

Comment:
Why the fixed height here? Is it to reinforce all course images have the same height? If so, I'd recommend doing:

1. setting  a ``max-height`` instead
2. using the $baseline variable (our base unit of vertical/horizontal spacing) to help calculate this value (e.g. ``max-height: ($baseline*18)``)
3. creating Sass variables that you define at the top of this file, including one that is for ``$course-image-height`` and referencing it here.